Cargo.toml

```diff
@@ -1,11 +1,8 @@
 [package]
+authors = ["Jake Goulding <jake.goulding@integer32.com>", "Carol (Nichols || Goulding) <carol.nichols@integer32.com>"]
+license = "MIT/Apache-2.0"
```

Comment:
`cargo add` doesn't preserve the incoming format of the TOML file. It's also "just" a third-party command, not one from the Rust project itself, so it doesn't have to represent a grand unified representation of TOML 😸.

I also use `cargo add`; my main quibble is with changing aspects unrelated to the same commit and/or PR. Things like that generally make digging through the git log much more difficult in the future.